src/_pytest/pastebin.py

```diff
@@ -67,8 +67,13 @@ def create_new_paste(contents):
 
     params = {"code": contents, "lexer": "python3", "expiry": "1week"}
     url = "https://bpaste.net"
-    response = urlopen(url, data=urlencode(params).encode("ascii")).read()
-    m = re.search(r'href="/raw/(\w+)"', response.decode("utf-8"))
+    try:
+        response = (
+            urlopen(url, data=urlencode(params).encode("ascii")).read().decode("utf-8")
+        )
+    except OSError as exc_info:  # urllib errors
+        response = str(exc_info)
```

Comment:
Would the exception info ever contain the URL?

Errors should get logged in any case probably. (not sure about non-zero exit code myself)

Reply:
*Probably* not, but I can't give a guarantee. So it's probably best to revise this.

Reply:
New commit 25dba82171dc93ee9d81b41e21609eda1f82846a addresses this